src/flows/components/FlowContextMenu.tsx

```diff
@@ -16,27 +16,47 @@ interface Props {
 }
 
 const FlowContextMenu: FC<Props> = ({id, name}) => {
-  const {remove} = useContext(FlowListContext)
+  const {remove, clone} = useContext(FlowListContext)
 
   const handleDelete = () => {
     event('delete_notebook')
     remove(id)
   }
 
+  const handleClone = () => {
+    event('clone_notebook')
+    clone(id)
+  }
+
   return (
-    <Context.Menu
-      icon={IconFont.Trash}
-      color={ComponentColor.Danger}
-      shape={ButtonShape.Default}
-      text={`Delete ${PROJECT_NAME}`}
-      testID={`context-delete-menu ${name}`}
-    >
-      <Context.Item
-        label="Confirm"
-        action={handleDelete}
-        testID={`context-delete-flow ${name}`}
-      />
-    </Context.Menu>
+    <Context>
+      <Context.Menu
+        icon={IconFont.Duplicate}
+        color={ComponentColor.Secondary}
+        shape={ButtonShape.Default}
+        text="Clone"
+        testID={`context-clone-menu ${name}`}
+      >
+        <Context.Item
```

Comment:
does this create a dropdown menu with a single item in it that says "clone" after clicking the "clone" button?

if so, we should not have that, and instead just clone it

Reply:
It does. I tried doing that prior to this but it doesn't look like the current context.menu or context support a button as a child, so i was getting some TS issues throwing a button in. I can try refactoring this to be the same pattern as the other parts of the app where there's an icon and clicking it will toggle that little context item text to select the action to perform

Reply:
i think you can ignore all the context stuff and through a button in there with an icon and a componentsize to do the trick